docs/developer/architecture/security/feature-registration.asciidoc

```diff
@@ -59,6 +59,11 @@ of features within the management screens.
 |See <<example-3-discover,Example 3>>
 |The set of subfeatures that enables finer access control than the `all` and `read` feature privileges. These options are only available in the Gold subscription level and higher.
 
+|`scope` (optional)
+|`string`
+|`["agnostic", "security"]`
```

Comment:
We've used the term "Space agnostic" to describe features, saved objects, etc. which do not obey space isolation. Using "agnostic" here to describe features which are allowed to be toggled within the space may be confusing.

```suggestion
|`["spaces", "security", "both"]`
```

Something like this might be easier to reason about. When `spaces` is chosen, then `privileges` must be null.

I'm open to other ideas as well, this was the first that came to mind.

Reply:
hmmm, maybe make it an array then instead of `string`? then we would assign scope: `['spaces', 'security']` to all exisitng features and just `['security']` to new ones, WDYT?

Reply:
I struggle to come with a better name for `both` 😅

Reply:
> hmmm, maybe make it an array then instead of string? then we would assign scope: ['spaces', 'security'] to all exisitng features and just ['security'] to new ones, WDYT?

Yeah I'd be happy with that approach too. It's cleaner than introducing a `both` option.